app/views/certificates/primary_certificate/_achievement_list.html.erb

```diff
@@ -40,14 +40,16 @@
 <% @community_groups.each do |group| %>
   <ul class="govuk-list ncce-activity-list ncce-activity-list--programme" role="list">
     <p class="govuk-body ncce-activity-list__title <%= add_group_complete_icon_class(current_user, group) %>">
-      <strong>Choose at least <%= group.required_for_completion.humanize %> activity</strong> to <%= group.title %>
+      <strong>Choose at least <%= group.required_for_completion.humanize %> activity</strong> to <%= group.title.downcase %>
     </p>
-    <%= render 'certificates/pathways/community_achievements',
-      programme_activities: split_community_pathway_activities_by_visibility(
-        group: group,
-        pathway_activities: @recommended_community_activities,
-      ),
-      community_achievements: @community_achievements
-    %>
+    <% group.order_programme_activities_for_user(current_user).each do |programme_activity| %>
```

Comment:
There are a lot of similarities between all of the X/_achievement_list.html.erb files, at some point we probably want to think of a reasonable abstraction for them. For now, I think we're best to stick to duplication